tests/lib/ocpp/v201/test_database_handler.cpp

```diff
@@ -53,7 +54,7 @@ TEST_F(DatabaseHandlerV201, KO1_FR27_DatabaseWithProfileData_UpdateProfile) {
 }
 
 TEST_F(DatabaseHandlerV201, KO1_FR27_DatabaseWithProfileData_InsertNewProfile) {
-    db_handler->insert_or_update_charging_profile(1, ChargingProfile{1, 1});
+    db_handler->insert_or_update_charging_profile(1, ChargingProfile{.id = 1, .stackLevel = 1});
     db_handler->insert_or_update_charging_profile(1, ChargingProfile{2, 1});
```

Comment:
Really like the change to being explicit when assigning struct fields; Can we do it on line 58 too for consistency?

```suggestion
    db_handler->insert_or_update_charging_profile(1, ChargingProfile{.id = 1, .stackLevel = 1});
    db_handler->insert_or_update_charging_profile(1, ChargingProfile{.id = 2, .stackLevel = 1});
```